packages/manager/apps/pci-rancher/src/components/Modal/DeleteModal/DeleteModal.component.tsx

```diff
@@ -40,12 +37,14 @@ const DeleteModal = ({
 }: DeleteModalProps) => {
   const { t } = useTranslation('listing');
   const [terminateText, setTerminateText] = useState('');
-  useTrackingPage(TrackingPageView.DeleteRancherModal);
-  const trackAction = useTrackingAction();
+
+  const { trackClick, trackCurrentPage } = useOvhTracking();
+
   const isButtonDisabled = TERMINATE_TEXT !== terminateText;
+  trackCurrentPage();
```

Comment:
**nitpick:** We have default track page triggered. Use that instead of calling `trackCurrentPage()` in all pages.